tardis/tardis_portal/templates/tardis_portal/index.html

```diff
@@ -13,9 +13,12 @@
     <h1>{{site_title|default:'MyTardis'}} Data Store</h1>
   </div>
   <p>
-    {% if not is_authenticated %}
```

Comment:
@carlohamalainen I shouldn't have pulled this as is, it looks like you removed the `is_authenticated` check. Can you have a look whether this change would work for you, too: https://github.com/mytardis/mytardis/pull/476 ?